presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcPageSink.java

```diff
@@ -170,7 +170,7 @@ public void abort()
             connection.rollback();
         }
         catch (SQLException e) {
-            throw new PrestoException(JDBC_ERROR, e);
+            // ignore exception from close
```

Comment:
i'd prefer to log warn here. If you're concerned about unnecessary logging this can be
```
log.warn("... failed: %s (full exception on DEBUG)", e);
log.debug(e, "....");
```

Reply:
```
// ignore exception from close
```
This is not exact, exception could be also raised by `connection.rollback();`